ansible/roles/schulcloud-server-core/templates/common-cartridge-deployment.yml.j2

```
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: common-cartridge-deployment
+  namespace: {{ NAMESPACE }}
+  labels:
+    app: common-cartridge
+    app.kubernetes.io/part-of: schulcloud-verbund
+    app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+    app.kubernetes.io/name: common-cartridge
+    app.kubernetes.io/component: server
+    app.kubernetes.io/managed-by: ansible
+    git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+    git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+spec:
+  replicas: {{ COMMON_CARTRIDGE_REPLICAS|default("1", true) }}
+  strategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxSurge: 1
+      #maxUnavailable: 1
+  revisionHistoryLimit: 4
+  paused: false
+  selector:
+    matchLabels:
+      app: common-cartridge
+  template:
+    metadata:
+      labels:
+        app: common-cartridge
+        app.kubernetes.io/part-of: schulcloud-verbund
+        app.kubernetes.io/version: {{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        app.kubernetes.io/name: common-cartridge
+        app.kubernetes.io/component: server
+        app.kubernetes.io/managed-by: ansible
+        git.branch: {{ SCHULCLOUD_SERVER_BRANCH_NAME }}
+        git.repo: {{ SCHULCLOUD_SERVER_REPO_NAME }}
+    spec:
+      securityContext:
+        runAsUser: 1000
+        runAsGroup: 1000
+        fsGroup: 1000
+        runAsNonRoot: true
+      containers:
+      - name: common-cartridge
+        image: {{ SCHULCLOUD_SERVER_IMAGE }}:{{ SCHULCLOUD_SERVER_IMAGE_TAG }}
+        imagePullPolicy: IfNotPresent
+        envFrom:
+        - configMapRef:
+            name: api-configmap
+        - secretRef:
+            name: api-secret
+        - configMapRef:
+            name: common-cartridge-configmap
+        - secretRef:
+            name: common-cartridge-secret
+        command: ['npm', 'run', 'nest:start:common-cartridge']
+        resources:
+          limits:
+            cpu: {{ COMMON_CARTRIDGE_CPU_LIMITS|default("2000m", true) }}
+            memory: {{ COMMON_CARTRIDGE_MEMORY_LIMITS|default("500Mi", true) }}
+          requests:
+            cpu: {{ COMMON_CARTRIDGE_CPU_REQUESTS|default("100m", true) }}
+            memory: {{ COMMON_CARTRIDGE_MEMORY_REQUESTS|default("50Mi", true) }}
```

To low request and limit for memory even for dev
```suggestion
          limits:
            cpu: {{ COMMON_CARTRIDGE_CPU_LIMITS|default("2000m", true) }}
            memory: {{ COMMON_CARTRIDGE_MEMORY_LIMITS|default("2GI", true) }}
          requests:
            cpu: {{ COMMON_CARTRIDGE_CPU_REQUESTS|default("100m", true) }}
            memory: {{ COMMON_CARTRIDGE_MEMORY_REQUESTS|default("150Mi", true) }}
```